src/legacy/core_plugins/kibana/public/discover/angular/doc_table/components/table_row.ts

```diff
@@ -59,18 +56,18 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
       onAddColumn: '=?',
       onRemoveColumn: '=?',
     },
-    link: function ($scope, $el) {
+    link: ($scope: any, $el: any) => {
       $el.after('<tr data-test-subj="docTableDetailsRow" class="kbnDocTableDetails__row">');
       $el.empty();
 
       // when we compile the details, we use this $scope
-      let $detailsScope;
+      let $detailsScope: any;
```

Comment:
We could provide our own type here extending `ng.IScope` and allow the properties we’re setting on the scope.

Reply:
Merci! Done!